job_definitions/smoke_tests.yml

```diff
@@ -6,13 +6,16 @@
     display-name: "Apps are up - {{ environment }}"
     project-type: pipeline
     description: |
-      Check that the five Digital Marketplace apps are up and running in the {{ environment|upper }} environment:
+      Check that the eight Digital Marketplace apps are up and running in the {{ environment|upper }} environment:
       <ul>
         <li>API</li>
         <li>Search API</li>
+        <li>Admin frontend</li>
+        <li>Briefs frontend</li>
+        <li>Brief-responses frontend</li>
         <li>Buyer frontend</li>
         <li>Supplier frontend</li>
-        <li>Admin frontend</li>
+        <li>User frontend</li>
```

Comment:
Should we include the router as an 'app'? The smoke tests certainly would fail without it 😸

Reply:
My only problem with this is that if we mention it here we should really mention it on all the functional tests.

Saying that we could add the router to the smoke tests 🤔

Reply:
Hmm maybe not though
I don't think we can get the url of a router app

Reply:
OK, makes sense.